src/components/fields/image-upload/image-review/image-editor/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ForwardedRef } from "react";
+
+export interface IImageEditorProps {
+	maxSizeInB?: number;
```

Comment:
is `maxSizeInB` actually used? and is it meant to be kb?

Reply:
I not sure. It didnt use when it exported

Reply:
it is being used in `toDataURLWithLimit` to do compression, so I guess the prop should be passed down from `image-review.tsx`